maxim/logger/fireworks/utils.py

```
@@ -0,0 +1,263 @@
+                
+        Returns:
+            Response: A structured response object that can be processed by
+                the parse_completion method for consistent logging.
+        """
+        response = ChatCompletionResponse(
+            id=f"streaming-response-{uuid.uuid4()}",
+            created=int(time.time()),
+            choices=[ChatCompletionResponseChoice(
+                index=0,
+                message=ChatMessage(role="assistant", content=content),
+                finish_reason="stop"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)